crates/shared/src/sources/uniswap_v3/pool_fetching.rs

```diff
@@ -36,6 +36,7 @@ pub trait PoolFetching: Send + Sync {
 /// Pool data in a format prepared for solvers.
 #[derive(Clone, Debug, Default, Eq, PartialEq, Serialize)]
 pub struct PoolInfo {
+    #[serde(skip_serializing)]
```

Comment:
Why?

Reply:
Because pool address is already serialized outside of this struct, so serializing the same value again makes it redundant.

Reply:
Could you add this as comment too. I think it will be hard to understand later.

Reply:
Added.